### ISSUE TITLE
none of the 'official' seed nodes work

### ISSUE THREAD

**Issue by akosmaroy** [z-classic/zclassic#105]
Hi,

I'm trying to run zclassic zcashd on an ubuntu box. I compiled from source from this github. I've edited 
~/.zclassic/zclassic.conf as per the documentation to contain:

```
addnode=dnsseed.zclassic.org
addnode=dnsseed.rotorproject.org
```

when I start zcashd, it doesn't seem to connect to any of these. when I start in a console, this is what I get:

```
Thank you for running a Zclassic node!
You're helping to strengthen the network and contributing to a social good :)

In order to ensure you are adequately protecting your privacy when using
Zclassic, please see <https://z.cash/support/security/index.html>.

           Block height | 0
            Connections | 0
  Network solution rate | 0 Sol/s

You are currently not mining.
To enable mining, add 'gen=1' to your zcash.conf and restart.

Since starting this node 3 minutes, 28 seconds ago:
- You have validated no transactions.

```

moreover:

```
$ zcash-cli getinfo
{
  "version": 1001051,
  "protocolversion": 170002,
  "walletversion": 60000,
  "balance": 0.00000000,
  "blocks": 0,
  "timeoffset": 0,
  "connections": 0,
  "proxy": "",
  "difficulty": 1,
  "testnet": false,
  "keypoololdest": 1506879165,
  "keypoolsize": 101,
  "paytxfee": 0.00000000,
  "relayfee": 0.00000100,
  "errors": ""
}
$ zcash-cli getpeerinfo
[
]

```

from the two nodes suggested to be put in the config file, one doesn't ping, the other gives a very strange FQDN:

```
$ ping dnsseed.zclassic.org
PING dnsseed.zclassic.org (197.12.8.45) 56(84) bytes of data.
From 197.13.3.22 (197.13.3.22) icmp_seq=3 Destination Host Unreachable
$ ping dnsseed.rotorproject.org
PING dnsseed.rotorproject.org (69.64.147.10) 56(84) bytes of data.
64 bytes from ash.parking.local (69.64.147.10): icmp_seq=1 ttl=236 time=185 ms

```

I wonder what is going wrong here.

more platform version info:

```
$ zcashd --version
Zcash Daemon version v1.0.10-1-c71b0bbb

In order to ensure you are adequately protecting your privacy when using
Zclassic, please see <https://z.cash/support/security/index.html>.

Copyright (C) 2009-2017 The Bitcoin Core Developers
Copyright (C) 2015-2017 The Zcash Developers
Copyright (C) 2015-2017 zdeveloper.org

This is experimental software.

Distributed under the MIT software license, see the accompanying file COPYING
or <http://www.opensource.org/licenses/mit-license.php>.

This product includes software developed by the OpenSSL Project for use in the
OpenSSL Toolkit <https://www.openssl.org/> and cryptographic software written
by Eric Young and UPnP software written by Thomas Bernard.
$ zcash-cli --version
Zcash RPC client version v1.0.10-1-c71b0bbb

In order to ensure you are adequately protecting your privacy when using
Zclassic, please see <https://z.cash/support/security/index.html>.

Copyright (C) 2009-2017 The Bitcoin Core Developers
Copyright (C) 2015-2017 The Zcash Developers
Copyright (C) 2015-2017 zdeveloper.org

This is experimental software.

Distributed under the MIT software license, see the accompanying file COPYING
or <http://www.opensource.org/licenses/mit-license.php>.

This product includes software developed by the OpenSSL Project for use in the
OpenSSL Toolkit <https://www.openssl.org/> and cryptographic software written
by Eric Young and UPnP software written by Thomas Bernard.
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 17.04
Release:	17.04
Codename:	zesty

```

**Comment by heyrhett** [comment]
Thank you, we need new seed nodes.  Does anyone have suggestions for what to use?

**Comment by lutsifer** [comment]
@akosmaroy 
I had same problem.

use this in the config:
addnode=dnsseed.zclassic.org:8133
addnode=dnsseed.rotorproject.org:8133
addnode=104.207.149.241:8133
addnode=45.76.4.70:8133
addnode=68.234.31.67:8133
addnode=52.14.116.113:8133
addnode=136.243.50.159:8133
addnode=88.99.199.93:8133
addnode=13.91.45.250:8133
addnode=94.130.11.10:8133
addnode=37.187.167.145:8133
addnode=5.9.111.40:8133
addnode=120.27.230.83:8133
addnode=138.197.199.86:8133
addnode=52.53.125.97:8133
addnode=198.100.147.192:8133
addnode=79.98.144.29:8133
addnode=62.163.150.178:8133
addnode=94.177.229.186:8133
addnode=78.94.32.196:8133
addnode=78.94.32.194:8133
addnode=60.170.218.192:8133
addnode=151.80.96.105:8133
addnode=139.59.230.86:8133
addnode=45.32.233.113:8133
addnode=45.32.234.100:8133
addnode=87.103.195.162:8133
addnode=58.9.134.10:8133
addnode=206.255.161.228:8133
addnode=89.212.102.22:8133
addnode=79.164.95.176:8133
addnode=78.8.88.180:8133
addnode=77.35.255.255:8133
addnode=197.245.248.194:8133
addnode=195.91.186.23:8133
addnode=138.68.48.247:8133
addnode=2.139.88.157:8133
addnode=35.202.93.102:8133
addnode=76.90.44.140:8133
addnode=107.143.247.252:8133
addnode=189.202.72.85:8133
addnode=66.119.109.246:8133

also define
rpcport=8232

**Comment by lutsifer** [comment]
make sure you open peer ports too

**Comment by BitCoinJedi** [comment]
Did adding the server nodes fix the problem? And do you literally put "define rpcport=8232" in the zcash.conf file?

**Comment by aribo** [comment]
@lutsifer how do you open peer ports?

**Comment by Mrwh0** [comment]
not define . just 
rpcport=8232

in .config file

**Comment by fredmeister77** [comment]
The testnet seed nodes do not appear to be working anymore either:
addnode=rotorproject.org
addnode=test-dnsseed.rotorproject.org
addnode=test-dnsseed.zclassic.org:8133
addnode=test-dnsseed.rotorproject.org:8133

Does anyone have recommendations for alternates?
(Let me know if this should be a separate issue).

**Comment by botio** [comment]
how to open peer port ?????

**Comment by Klintistwood** [comment]
Hello @akosmaroy 

did you solve the issue? I have exactly the same problem as you, using the same version too but changing zclassic.conf with the above suggestion did not help :(

thanks

**Comment by akosmaroy** [comment]
no, I haven't

2018-01-01 12:43 GMT+01:00 Klintistwood <notifications@github.com>:

**Comment by Klintistwood** [comment]
I tried this https://www.yougetsignal.com/tools/open-ports/ to see if port 8133 was open and it's closed. I checked my routers to open this port but it doesn't change anything. I directly connected to my modem (bypassing al routers) and port is still closed so there is nothing I can do to open it. I also tried my 4G connection with my smartphone and port is closed too.

Among the ones for who it's working here, can you check if this port is open for you? If you get the sync to work with the port closed, it means that the port is not the issue .

Thanks

**Comment by matthewjamesr** [comment]
@akosmaroy We are uping new nodes within 24 hours. I only have one almost fully sync'd, please add it.

`addnode=na1.zclassic.org`

eu1. && as1. will be up tomorrow.

**Comment by lk29** [comment]
try Zclassic | ZCL Network Peers from 
https://www.coinexchange.io/network/peers/ZCL

**Comment by papercheck** [comment]
@Klintistwood, I don't think it is enough to open the port on the firewall/router. I had to take one extra step and add **port=8133** to the zclassic.conf file. My file looks something like this:

    rpcuser=change-this
    rpcpassword=change-this
    rpcallowip=127.0.0.1
    rpcport=8232
    port=8133  
    addnode=50.112.137.36
    addnode=188.166.136.203
    addnode=159.89.198.93
    addnode=zcl.suprnova.cc
    addnode=dnsseed.rotorproject.org
    addnode=dnsseed.zclassic.org

Once I was able to make that change, I tested my port with https://www.yougetsignal.com/tools/open-ports/ and I can see the port is now open for my server 98.179.250.220.

Prior to this, when I ran `./src/zcash-cli getnetworkinfo` the port was listed as 8033. I am not sure if that is the default port, but it was not set to 8133.

I hope this helps!

**Comment by akosmaroy** [comment]
I wonder why this issue was closed. so far it still doesn't work for me.

for one, the two nodes listed in the User Guide at https://github.com/z-classic/zclassic/wiki/1.0-User-Guide don't even resolve:

```
$ ping dnsseed.zclassic.org
ping: dnsseed.zclassic.org: Name or service not known
$ ping dnsseed.rotorproject.org
ping: dnsseed.rotorproject.org: Name or service not known
```

the nodes suggested by @matthewjamesr don't resolve either:

```
$ ping na1.zclassic.org
ping: na1.zclassic.org: Name or service not known
$ ping eu1.zclassic.org
ping: eu1.zclassic.org: Name or service not known
$ ping as1.zclassic.org
ping: as1.zclassic.org: Name or service not known
```

should one really add IP addresses into this list?

**Comment by Klintistwood** [comment]
for me it worked, I added new seeds, it took some time before starting but in the end it started the sync. I did not have to change anything but the seeds.

**Comment by Robi026** [comment]
What nodes we should now use for the testnet?

**Comment by Mrwh0** [comment]
for testnet use this one https://github.com/nxsofsys/zclassic/commits/new_genesis

**Comment by kiahmed** [comment]
none of this worked.is there any active testnode ?
